go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/asaskevich/EventBus v0.0.0-20180315140547-d46933a94f05 // indirect
 	github.com/azr/backoff v0.0.0-20160115115103-53511d3c7330 // indirect
 	github.com/briandowns/openweathermap v0.0.0-20180804155945-5f41b7c9d92d
-	github.com/bwmarrin/discordgo v0.19.0
+    github.com/bwmarrin/discordgo v0.23.1
```

Comment:
Just a heads up: you don't have to edit these files manually. You can run `go get github.com/bwmarrin/discordgo` to add a new dependency, which will updates these files for you. Similarly to `gofmt` you can now run `go mod tidy` which should clean them up for you!

Reply:
Ah great! Thank you! I'll keep this in mind 👍